### ISSUE TITLE
Node class DeepCover::Node::Block incorrectly defined

### ISSUE THREAD

**Issue by ragesoss** [deep-cover/deep-cover#11]
I've got some debug info from running my rails test suite with deep-cover. [Source code here](https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/lib/importers/revision_score_importer.rb).

```
Node class DeepCover::Node::Block incorrectly defined
Attempting to continue, but this node will not be handled properly

Source:
revisions&.select { |rev| rev.wiki_id == @wiki.id }
Original exception:
#<TypeError: Invalid children types for DeepCover::Node::Block(type: block): [[(csend
  (lvar[Variable] :revisions)
  (safe_send[Send] nil :select)), [DeepCover::Node::SendWithBlock, DeepCover::Node::SuperWithBlock]]]>
DeepCover is getting confused with the file /home/sage/WikiEduDashboard/lib/importers/revision_score_importer.rb and it won't be instrumented.
Please report this error and provide the source code around the following:
/home/sage/WikiEduDashboard/lib/importers/revision_score_importer.rb:29: syntax error, unexpected '{', expecting ')'
...ov[2575][24]+=1;_temp=_temp)) { |rev| $_cov[2575][30]+=1;_te...
...                               ^
/home/sage/WikiEduDashboard/lib/importers/revision_score_importer.rb:29: syntax error, unexpected '}', expecting ')'
...ov[2575][28]+=1;_temp=_temp)) });$_cov[2575][29]+=1;_temp=_t...
...                               ^
/home/sage/WikiEduDashboard/lib/importers/revision_score_importer.rb:37: syntax error, unexpected keyword_end, expecting ')'
  end);$_cov[2575][49]+=1;_temp=_temp))
     ^
/home/sage/WikiEduDashboard/lib/importers/revision_score_importer.rb:37: syntax error, unexpected ')', expecting keyword_end
/home/sage/WikiEduDashboard/lib/importers/revision_score_importer.rb:106: dynamic constant assignment
  DELETED_REVISION_ERRORS = ((_temp=(%w[TextDeleted Revi...
                           ^
/home/sage/WikiEduDashboard/lib/importers/revision_score_importer.rb:138: dynamic constant assignment
  WP10_WEIGHTING = ((_temp=({ 'FA'    => 100,
                  ^
/home/sage/WikiEduDashboard/lib/importers/revision_score_importer.rb:164: syntax error, unexpected end-of-input, expecting keyword_end
```

**Comment by marcandre** [comment]
Oh, right. I implemented `&.` but forgot about it with a block...
Thanks for the report, should be fixed quickly.
At least diagnostics work correctly 😸

**Comment by marcandre** [comment]
I think I got it!
Released as 0.1.13.
